_tests/integration/reactnative_test.go

```diff
@@ -147,81 +162,90 @@ var sampleAppsReactNativeSubdirVersions = []interface{}{
+    env_key: WORKDIR
     type: selector
     value_map:
-      project/android:
-        title: Module
-        summary: Modules provide a container for your Android project's source code,
-          resource files, and app level settings, such as the module-level build file
-          and Android manifest file. Each module can be independently built, tested,
-          and debugged. You can add new modules to your Bitrise builds at any time.
-        env_key: MODULE
-        type: user_input
+      project:
+        title: The root directory of an Android project
+        summary: The root directory of your Android project, stored as an Environment
+          Variable. In your Workflows, you can specify paths relative to this path.
+          You can change this at any time.
+        env_key: PROJECT_LOCATION
+        type: selector
         value_map:
-          app:
-            title: Variant
-            summary: Your Android build variant. You can add variants at any time,
-              as well as further configure your existing variants later.
-            env_key: VARIANT
-            type: user_input_optional
+          project/android:
+            title: Module
+            summary: Modules provide a container for your Android project's source
+              code, resource files, and app level settings, such as the module-level
+              build file and Android manifest file. Each module can be independently
+              built, tested, and debugged. You can add new modules to your Bitrise
+              builds at any time.
+            env_key: MODULE
+            type: user_input
             value_map:
-              "":
-                title: Project or Workspace path
-                summary: The location of your Xcode project or Xcode workspace files,
-                  stored as an Environment Variable. In your Workflows, you can specify
-                  paths relative to this path.
-                env_key: BITRISE_PROJECT_PATH
-                type: selector
+              app:
+                title: Variant
+                summary: Your Android build variant. You can add variants at any time,
+                  as well as further configure your existing variants later.
+                env_key: VARIANT
+                type: user_input_optional
                 value_map:
-                  project/ios/SampleAppsReactNativeAndroid.xcodeproj:
-                    title: Scheme name
-                    summary: An Xcode scheme defines a collection of targets to build,
-                      a configuration to use when building, and a collection of tests
-                      to execute. Only shared schemes are detected automatically but
-                      you can use any scheme as a target on Bitrise. You can change
-                      the scheme at any time in your Env Vars.
-                    env_key: BITRISE_SCHEME
+                  Debug:
```

Comment:
NOTE: the android project has a single variant (`Debug`) or it is an issue in the options concatenation?

Reply:
Neither, we have a default variant and only that is listed as an option value.